API/RestSharpAPI/Steps/BookingsApiSteps.cs

```diff
@@ -0,0 +1,283 @@
+using RestSharpApi.Data;
+using RestSharpApi.Hooks;
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using TechTalk.SpecFlow;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using NUnit.Framework;
+using Bookings;
+using Notification;
+using User;
+using Video;
+using System.Collections.Generic;
+
+namespace RestSharpApi.Steps
+{
+    [Binding]
+    public class BookingsApiSteps : RestSharpHooks
+
+    {
+        private static BookingsApi BookingApiService;
+        protected static ScenarioContext _scenarioContext;
+        private static string HearingScenarionKey = "Hearing";
+
+        [When(@"I check BookingsApi health")]
+        public void WhenICheckBookingsApiHealth()
+        {
+            throw new PendingStepException();
```

Comment:
this step is not implemented. Is it intentional?

Reply:
For now.  It is required and will be added at some point.